default/stringtables/de.txt

```diff
@@ -320,16 +320,16 @@ SD_MARK1_DESC
 Einfache Fregatte mit Massenkatapult. Kann nur auf Kolonien mit einem [[buildingtype BLD_SHIPYARD_BASE]] gebaut werden.
 
 SD_LARGE_MARK_1
-Zerstörer (M1)
+Kreuzer (M)
```

Comment:
Why `Kreuzer (M)` and not `Kreuzer Ms` or more similar to the english equivalent? https://github.com/freeorion/freeorion/blob/master/default/stringtables/en.txt#L390

Reply:
Probably because it is not clear what `Ms` or `Lz` stands for.  I just recently saw the same translation, but didn't fix this because the meaning of the suffix was unclear.

Reply:
@MatGB *poke* You made those translation changes.

Reply:
Probably those designs should be renamed in english as well then...

Reply:
I would vote for adding an explanation of the naming scheme somewhere, rather than just doing away with it.

Reply:
The naming scheme does seem like it probably works well enough in German, yes?  's' still for Standard [Armor], z still for Zortrium, etc.?   I expect that for some parts the naming scheme would have to use different initials between English and German, but that still doesn't really seem like a problem to me.